### PR TITLE
Grafana HA

### PR DESCRIPTION
**Pull request by saurabh3460**

### CODE REVIEW

File: monitoring/deployment/postgresql-deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
```
Comment (Hashfyre) [inline monitoring/deployment/postgresql-deploy.yaml]: This should be under the folder structure,
/monitoring/grafana-postgres/statefulset.yaml

File: monitoring/deployment/postgresql-deploy.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  selector:
+    app: postgresql
+  ports:
+    - protocol: TCP
+      port: 5432
+      targetPort: 5432
+  type: ClusterIP
+---
+apiVersion: apps/v1
+kind: Deployment
```
Comment (Hashfyre) [inline monitoring/deployment/postgresql-deploy.yaml]: Convert to a `statefulset`

**Review by Hashfyre — CHANGES_REQUESTED** [review]

File: monitoring/deployment/postgresql-deploy.yaml (hunk 1 above)
Comment (saurabh3460) [reply monitoring/deployment/postgresql-deploy.yaml]: made changes

File: monitoring/deployment/postgresql-deploy.yaml (hunk 2 above)
Comment (saurabh3460) [reply monitoring/deployment/postgresql-deploy.yaml]: made sts

File: monitoring/grafana-postgres/statefulset.yaml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        name: data
+      spec:
+        accessModes: ["ReadWriteOnce"]
+        storageClassName: "gp2"
+        resources:
+          requests:
+            storage: 1Gi
+---
+apiVersion: batch/v1
+kind: Job
```
Comment (Hashfyre) [inline monitoring/grafana-postgres/statefulset.yaml]: Move to it's own manifests.

**Review by Hashfyre — CHANGES_REQUESTED** [review]

**Review by Hashfyre — APPROVED** [review]
LGTM

**Pull request merged**